### PR TITLE
Add policy edit ( information and roles modals)

### PR DESCRIPTION
**Pull request by lgalis**
Implements https://projects.engineering.redhat.com/browse/SSP-756

Screenshot from 2019-10-11 14-56-28

Screenshot from 2019-10-11 14-56-05

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -29,6 +29,10 @@
   }
 }
 
+.select-style {
```
Comment (Hyperkid123) [inline src/App.scss]: I do understand why this exists but i am not sure if this is the way forward (right now there is no other option...). This is problem with forms inside modals. I will bring this UP with Susan and try to figure out some future proof solution.

File: src/smart-components/group/policy/policy-actions/edit-policy-roles.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    if (policy) {
+      const policy_data = {
+        name: policy.name,
+        group: policy.group.uuid,
+        roles: selectedRoles.map(role => role.value)
+      };
+      return postMethod ? updatePolicy(policy.uuid, policy_data).then(() => postMethod()).then(() => push(closeUrl)) :
+        updatePolicy(policy.uuid, policy_data).then(() => push(closeUrl));
+    }
+    else {
```
Comment (Hyperkid123) [inline src/smart-components/group/policy/policy-actions/edit-policy-roles.js]: no need fo the else branch. Main branch has return statement.

File: src/App.scss (hunk 1 above)
Comment (lgalis) [reply src/App.scss]: Sounds good, thank you!

Comment (lgalis) [reply src/App.scss]: @Hyperkid123 - my next PR will replace the select with a compact table - that is in the UX mocks

File: src/smart-components/group/policy/policy-actions/edit-policy-roles.js (hunk 2 above)
Comment (lgalis) [reply src/smart-components/group/policy/policy-actions/edit-policy-roles.js]: @Hyperkid123  - Updated

**Pull request merged**